instance/tests/models/test_redis_mixin.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2019 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+OpenEdXInstance Redis Database Mixins - Tests
+"""
+
+# Imports #####################################################################
+
+from unittest.mock import patch
+
+import ddt
+import yaml
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from instance.models.openedx_instance import OpenEdXInstance
+from instance.tests.base import TestCase
+from instance.tests.models.factories.openedx_appserver import make_test_appserver
+from instance.tests.models.factories.openedx_instance import OpenEdXInstanceFactory
+from registration.models import BetaTestApplication
+
+
+# Tests #######################################################################
+
+@ddt.ddt
+class OpenEdXThemeMixinTestCase(TestCase):
```

Comment:
@swalladge @gabor-boros -- these tests look unrelated to this PR? Do we need to merge latest master or something?

Reply:
@pomegranited ah yes this looks like a copy/paste issue.  :P  I can remove this file altogether.